src/v/cluster/scheduling/partition_allocator.cc

```diff
@@ -81,6 +81,92 @@ allocation_constraints partition_allocator::default_constraints() {
     return req;
 }
 
+bool guess_is_memory_group_aware_memory_limit() {
+    // Here we are trying to guess whether `topic_memory_per_partition` is
+    // memory group aware. Originally the property was more like a guess of how
+    // much memory each partition would statically use. It was first set to 10
```

Comment:
Pretty sure it was 1 MiB? We raised it, not lowered it IIRC.

Reply:
Ah now that you say it that rings a bell. I misremembered.